### PR TITLE
Run Markdownlint and Prettier

### PR DESCRIPTION
**Pull request by nschonni**
Ran Markdownlint to clean up a few things and flag some things like uneven heading levels, then ran Prettier for the remaining spaces.
There was one external image from another post that I pulled into the repo instead of hot-linking

### CODE REVIEW

**Comment by zkoppert** [comment]
Thanks again @nschonni! Would you be interested in taking a pass at writing a github action workflow so we could always keep things looking nice by running the markdownlint and prettier? Or did you do a significant amount of filtering those results to come to these changes?

**Comment by nschonni** [comment]
Do you just want the actions, or a light package.json with the dependencies to go with it? I have the `.markdownlint-cl2.jsonc` still sitting in the repo, I just didn't push it with ths

**Comment by zkoppert** [comment]
Both would be great!

**Comment by nschonni** [comment]
OK, I added them to this PR in separate commits, but they could always get split out

**Comment by zkoppert** [comment]
Thanks! Looks great overall and I'll give a proper review tomorrow!

File: .markdownlint-cli2.jsonc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "config": {
+    "default": true,
+    "MD013": false, // Ignore line-length
+    "MD045": false, // Ignore alt text requirement for images
```
Comment (nschonni) [inline .markdownlint-cli2.jsonc:5]: This should probably be a TODO to fix later, but the current ones didn't have a text either

Comment (zkoppert) [reply .markdownlint-cli2.jsonc:5]: https://github.com/github/github-ospo/issues/18

**Review by zkoppert — APPROVED** [review]
🎉 Awesome improvements to keep things clean and well formatted! Thanks again @nschonni !

**Pull request merged**